stellarphot/settings/tests/test_models.py

```diff
@@ -29,3 +33,43 @@ def test_create_invalid_values(bad_one):
     bad_settings[bad_one] = -1
     with pytest.raises(ValidationError, match=bad_one):
         ApertureSettings(**bad_settings)
+
+
+DEFAULT_EXOPLANET_SETTINGS = dict(
+    epoch=Time(0, format="jd"),
+    period=0 * u.min,
+    identifier="",
+    coordinate=SkyCoord(
+        ra="00:00:00.00", dec="+00:00:00.0", frame="icrs",
+        unit=("hour", "degree")
+    ),
+    depth=0,
+    duration=0 * u.min,
+)
+
+
+def test_create_exoplanet_correctly():
+    planet = Exoplanet(**DEFAULT_EXOPLANET_SETTINGS)
+    assert planet.epoch == DEFAULT_EXOPLANET_SETTINGS["epoch"]
+    assert u.get_physical_type(planet.period) == "time"
+    assert planet.identifier == DEFAULT_EXOPLANET_SETTINGS["identifier"]
+    assert planet.coordinate == DEFAULT_EXOPLANET_SETTINGS["coordinate"]
+    assert planet.depth == DEFAULT_EXOPLANET_SETTINGS["depth"]
+    assert u.get_physical_type(planet.duration) == "time"
+
+
+@pytest.mark.parametrize("bad_one", [
```

Comment:
Usually `parametrize` is used to when you want to test with multiple values rather than a single value. I could see going two ways here:

1. Test two values, like in the example you mentioned [here](https://github.com/feder-observatory/stellarphot/pull/235#issuecomment-1870454532). That maybe looks like `@pytest.mark.parametrize('bad_one', ('period', 'duration'))` and then in the test set the value ot something bad like in the example.
2. Test with one set of settings in which both duration and period are bad (inappropriate units) and make sure the error message has the right number of `ValidationErrors`. See [this test](https://github.com/feder-observatory/stellarphot/blob/main/stellarphot/tests/test_core.py#L52) for an example of that approach.

Either way, you should be able to start with a copy of the default exoplanet and modify values of that rather than reproducing all the settings.

Reply:
So I redid the tests and just have a question about the order of operation for the tests. Does this pytest run the validation tests that are in the exoplanet class that check the physical units of duration and period? When I gave period and duration units of cm it would pass this pytest.

Reply:
It should run the validation checks and then raise a single `ValidationError` (rather than several `ValueError`s) with a summary of the errors it encountered.